e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -244,52 +271,143 @@ describe("scenarios > search", () => {
     });
 
     describe("search filters", () => {
-      typeFilters.forEach(
-        ({ label, sidebarLabel, filterName, resultInfoText }) => {
-          it(`should filter results by ${label}`, () => {
-            cy.visit("/");
-
-            cy.findByTestId("search-bar-filter-button").click();
-            getSearchModalContainer().within(() => {
-              cy.findByText(label).click();
-              cy.findByText("Apply all filters").click();
+      describe("type filters", () => {
+        typeFilters.forEach(
+          ({ label, sidebarLabel, filterName, resultInfoText }) => {
+            it(`should filter results by ${label}`, () => {
+              cy.visit("/");
+
+              cy.findByTestId("search-bar-filter-button").click();
+              getSearchModalContainer().within(() => {
+                cy.findByText(label).click();
+                cy.findByText("Apply all filters").click();
+              });
+
+              getSearchBar().clear().type("e{enter}");
+              cy.wait("@search");
+
+              cy.url().should("include", `type=${filterName}`);
+
+              cy.findAllByTestId("result-link-info-text").each(result => {
+                cy.wrap(result).should("contain.text", resultInfoText);
+              });
+
+              cy.findAllByTestId("type-sidebar-item").should("have.length", 2);
+              cy.findByTestId("type-sidebar").within(() => {
+                cy.findByText(sidebarLabel).should("exist");
+              });
             });
+          },
+        );
+      });
 
-            getSearchBar().clear().type("e{enter}");
-            cy.wait("@search");
+      describe("verified filter", () => {
+        it("should filter only for `Verified` assets", () => {
+          cy.visit("/");
 
-            cy.url().should("include", `type=${filterName}`);
+          cy.findByTestId("search-bar-filter-button").click();
+          getSearchModalContainer().within(() => {
+            cy.findByText("Only verified items").click();
+            cy.findByText("Apply all filters").click();
+          });
 
-            cy.findAllByTestId("result-link-info-text").each(result => {
-              cy.wrap(result).should("contain.text", resultInfoText);
-            });
+          getSearchBar().clear().type("Orders{enter}");
+          cy.wait("@search");
+
+          cy.url().should("include", `verified=true`);
+
+          const verifiedItem = cy.findByTestId("search-result-item");
+          verifiedItem.within(() => {
+            cy.findByLabelText("verified icon").should("exist");
+          });
```

Comment:
If there is only one item inside `within` block we could make the syntax more concise by chaining the function call instead

```suggestion
          verifiedItem.findByLabelText("verified icon").should("exist");
```